src/routes/current-studies/StudyCard.svelte

```diff
@@ -0,0 +1,141 @@
+*/
+
+import { createEventDispatcher } from 'svelte';
+import StudyCard from '../../components/study-card/StudyCard.svelte';
+import StudyCardHeader from '../../components/study-card/Header.svelte';
+import Button from '../../components/Button.svelte';
+import Dialog from '../../components/Dialog.svelte';
+
+export let joined = false;
+export let imageSrc;
+export let title = "Untitled Study";
+export let author = "Author Unknown";
+export let description = "no description.";
+export let endDate;
+export let joinedDate;
+export let dataCollectionDetails;
+export let privacyPolicyLink;
+export let tags;
+export let detailsDirectName;
+export let detailsDirectLink;
+export let joinStudyConsentNotice;
+export let leaveStudyConsentNotice;
+
+const dispatch = createEventDispatcher();
+
+let joinModal = false;
+
+function triggerJoinEvent() {
+    // send CTA click event to parent.
+    dispatch('cta-clicked');
+    joinModal = true;
+}
+
+</script>
+
+<StudyCard
+    {joined}
+    on:join={triggerJoinEvent}
+    on:leave={triggerJoinEvent}
+    {endDate}
+    {joinedDate}
+    {dataCollectionDetails}
+    {tags}
+    {imageSrc}
+    {privacyPolicyLink}>
+        <span slot="name">{title}</span>
+        <span slot="author">{author}</span>
+        <p slot="description">
+        {description}
+        </p>
+        <div slot="details" style="display: {(detailsDirectName !== undefined && detailsDirectLink !== undefined) ? 'auto' : 'none'};">
+        {#if detailsDirectName && detailsDirectLink}
+            Full study details can be found on the
+            <a href={detailsDirectLink}>{detailsDirectName}</a>
+        {/if}
+        </div>
+</StudyCard>
+
+{#if joinModal}
+    <Dialog
+    on:dismiss={() => {
+        joinModal = false;
+    }}>
+    <!-- override the dialog Zilla font setting. -->
+    <div slot="title">
+        {#if !joined}
+        <StudyCardHeader {endDate}>
+            <img
+            slot="study-icon"
+            class="study-card-image"
+            width="60"
+            alt="study icon"
+            src={imageSrc || "/default-study-icon.png"} />
+            <span slot="study-name">{title}</span>
+            <span slot="study-author">{author}</span>
+        </StudyCardHeader>
+        {:else}Leave this Study?{/if}
+    </div>
+    <div style="padding-bottom: 90px;" slot="body">
+        {#if !joined}
+        {#if joinStudyConsentNotice}
+            {@html joinStudyConsentNotice}
+        {:else}
+        <!-- default "join study" consent notice -->
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+            enim ad minim veniam, ut enim ad minim veniam, ut enim ad minim
+            veniam.
+            </p>
+            <p>TBD approach based on explorations with SimplySecure.</p>
+        {/if}
+        {:else}
+        {#if leaveStudyConsentNotice}
+            {@html leaveStudyConsentNotice}
```

Comment:
Hm, according to https://svelte.dev/tutorial/html-tags `@html` doesn't do any sanitization, not sure you need to do anything right now but if we start delivering from remote we'll need to consider this.

Reply:
For sure. Mind filing an issue for this?